podcasts/Common SwiftUI/Toast/ToastViewModel.swift

```diff
@@ -0,0 +1,75 @@
+import Foundation
+
+protocol ToastCoordinator: AnyObject {
```

Comment:
Do you think we can name this `ToastDelegate`? Coordinator always reminds me of the [coordinator pattern](https://www.hackingwithswift.com/articles/71/how-to-use-the-coordinator-pattern-in-ios-apps).

Reply:
That makes more sense that coordinator, done here: 4c0e5e522a6ab00bd66faccab60f6d78ab783b15